Gruntfile.js

```diff
@@ -1,20 +1,31 @@
 'use strict';
 module.exports = function (grunt) {
+
+    // Set the video.js version
+    var vjsVersion = grunt.file.readJSON('package.json').version;
+
     // Project configuration.
     grunt.initConfig({
         shell: {
             generateJSON: {
-                command: 'jsdoc --configure ./conf.json ./video.js/src/js > cumulative.json'
+                command: function(version) {
+                    // Generate API docs based on correct git tag.
+                    return 'cd video.js && git checkout v' + version + ' && cd ../ && jsdoc --configure ./conf.json ./video.js/src/js > cumulative.json';
+                }
             },
             cloneVideoJS: {
-                // Once 5.0 is in stable the line below should be use instead
-                // command: 'rm -rf ./video.js && git clone -b stable --single-branch https://github.com/videojs/video.js.git'
-                command: 'rm -rf ./video.js && git clone -b stable --single-branch https://github.com/videojs/video.js.git'
+                command: 'rm -rf ./video.js && git clone https://github.com/videojs/video.js.git'
             },
+            gitPull: {
+                // Used if video.js has alredy been cloned and we want the latest code to work with.
+                command: 'cd video.js && git pull origin master'
```

Comment:
Come to think of it I think this should be `git fetch`. Should it be `git fetch --tags`?